smdk4412-common/common-vendor-blobs.mk

```diff
@@ -18,8 +18,10 @@ PRODUCT_PACKAGES += \
 	libUMP \
 	libfimc
 
+ifeq ($(filter n5100 n5110,$(TARGET_DEVICE)),)
```

Comment:
This can be avoided by using the right order of inheritance.
For PRODUCT_COPY_FILES, the first rule takes precedence, so if you make sure kona-common is inherited before smdk4412-common, the binary from kona will be used.

I have an entire device based on this, https://github.com/CyanogenMod/android_device_sony_togari_gpe